### PR TITLE
Moving btr-api into subdir

### PR DESCRIPTION
**Pull request by hfekete**
Moving btr-api into subdir

### CODE REVIEW

File: btr-api/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![License](https://img.shields.io/badge/License-BSD%203%20Clause-blue.svg)](LICENSE)
+[![License](https://img.shields.io/badge/License-BSD%203%20Clause-blue.svg)](../LICENSE)
```
Comment (kialj876) [inline btr-api/README.md:1]: was this on purpose?

Comment (hfekete) [reply btr-api/README.md:1]: Yes, Thor suggested that we leave license in the root dir.

**Review by kialj876 — APPROVED** [review]

File: migrations/__pycache__/env.cpython-311.pyc
Comment (thorwolpert) [inline migrations/__pycache__/env.cpython-311.pyc:1]: pyc files should be ignored and deleted

File: tests/unit/resources/__pycache__/test_submissions.cpython-311-pytest-7.4.0.pyc
Comment (thorwolpert) [inline tests/unit/resources/__pycache__/test_submissions.cpython-311-pytest-7.4.0.pyc:1]: all of `__pycache__` should be ignored and deleted

**Review by thorwolpert — COMMENTED** [review]
just a couple of suggestions.

File: .env
[the remark this answers is not in the document]
Comment (hfekete) [reply .env:1]: Readded it. I prefer keeping .env files out of git though, as they can have sensitive data in them, and sometimes ppl can push them accidentally.

File: migrations/__pycache__/env.cpython-311.pyc
Comment (hfekete) [reply migrations/__pycache__/env.cpython-311.pyc:1]: Can you point if there is any left, I dont see any in my PR (other then deletes) ?
I purged the __pycache__ that was originally in the repo.

File: tests/unit/resources/__pycache__/test_submissions.cpython-311-pytest-7.4.0.pyc
Comment (hfekete) [reply tests/unit/resources/__pycache__/test_submissions.cpython-311-pytest-7.4.0.pyc:1]: Can you point if there is any left, I dont see any in my PR (other then deletes) ?
I purged the __pycache__ that was originally in the repo.

**Pull request merged**